cocos/rendering/custom/define.ts

```diff
@@ -2589,6 +2589,11 @@ export function SetLightUBO (
         buffer[index++] = direction.x;
         buffer[index++] = direction.y;
         buffer[index] = direction.z;
+
+        buffer[offset + UBOForwardLight.LIGHT_BOUNDING_SIZE_VS_OFFSET + 0] = 0;
+        buffer[offset + UBOForwardLight.LIGHT_BOUNDING_SIZE_VS_OFFSET + 1] = 0;
+        buffer[offset + UBOForwardLight.LIGHT_BOUNDING_SIZE_VS_OFFSET + 2] = 0;
+        buffer[offset + UBOForwardLight.LIGHT_BOUNDING_SIZE_VS_OFFSET + 3] = spotLight.spotAngleAttenuationStrength;
```

Comment:
- Why need to use float4? Can just use float?
- If float4 is needed, can just offset 0?

Reply:
It reuses current LIGHT_BOUNDING_SIZE_VS_OFFSET.  xyzw are all not used by Spotlight. xyz is used by ranged directional light. So I choose w to store the value, to avoid future potential collision.

Reply:
Got it.